packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -322,6 +327,14 @@ const _BaseLink: React.ForwardRefRenderFunction<BladeElementRef, BaseLinkProps>
     }
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleBlur = (event: any): void => {
+    onBlurInteraction();
+    if (onBlur) {
+      onBlur(event);
+    }
```

Comment:
```suggestion
onBlur?.(event);
```

Reply:
I like the explicit conditional block at some places for readability 😅